packages/core-data/src/queried-data/reducer.js

```diff
@@ -138,6 +140,37 @@ const queries = flowRight( [
 	);
 } );
 
+/**
+ * Reducer tracking queries state.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Next state.
+ */
+const queries = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case 'RECEIVE_ITEMS':
+			return receiveQueries( state, action );
+		case 'REMOVE_ITEMS':
+			const newState = { ...state };
+			const removedItems = action.items.reduce( ( result, item ) => {
+				result[ item ] = true;
```

Comment:
`item` is an object isn't it?

Reply:
No, it's an int. `action.items` is `[2, 12, 34]`

Reply:
That doesn't match the type signature of `removeItems`.

It should be `itemIds` then.

Reply:
good point.